hugo.toml

```diff
@@ -18,11 +18,23 @@ title = 'MaxMind'
     tabWidth = 4
 
 [menus]
-  [[menus.main]]
+  [[menus.minfraud]]
     name = 'minFraud'
     pageRef = '/minfraud'
     weight = 10
-  [[menus.main]]
+  [[menus.geoip]]
     name = 'GeoIP'
     pageRef = '/geoip'
-    weight = 20
+    weight = 10
+  [[menus.general]]
```

Comment:
Is the idea that we'll move this page into a different section in the nav?

Reply:
Yep, instead of repeating the links in the GeoIP and minFraud sections